File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int flag=0,n=nums.size(),choice=1;
        for(int i=0;i<n-1;i++){
            if(nums[i]>nums[i+1]){
                if(choice==1){
                    choice--;
                }
                else{
                    return false;
                }
            }
        }
        if(choice==0){
            return nums[0]>=nums[n-1];
        }
        return true;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        cnt=0
        n=len(nums)
        for i in range(n):
            if nums[i]>nums[(i+1)%n]:
                cnt+=1
            if cnt>1:
                return False
        return True
